Document/0x05e-Testing-Cryptography.md

```diff
@@ -273,6 +296,12 @@ Hook cryptographic methods and analyze the keys that are being used. Monitor fil
 
 - [#nelenkov] - N. Elenkov, Android Security Internals, No Starch Press, 2014, Chapter 5.
 
+#### Cryptography references
+- https://android-developers.googleblog.com/2016/06/security-crypto-provider-deprecated-in.html
+- https://android-developers.googleblog.com/2018/03/cryptography-changes-in-android-p.html
+- https://www.hex-rays.com/products/ida/
+- https://android-developers.googleblog.com/2016/06/android-changes-for-ndk-developers.html
```

Comment:
Can you give the references a title in the format [title](URL)

Reply:
👍